install_tools.sh

```diff
@@ -35,7 +35,7 @@ install_on_device() {
 
     # copy selected scriptlets over to the device
     # and then move them somewhere in the device's PATH
-    DEVICE_SCRIPTLETS=(retry check_for_packages_complete wait_for_packages_complete install_packages)
+    DEVICE_SCRIPTLETS=(retry check_for_packages_complete wait_for_packages_complete install_packages clean_machine)
```

Comment:
Looking at the name of the script, can I suggest renaming it to something that suggests it cleans traces of Checkbox, rather than cleaning a machine generally?

Reply:
I expect this script to be updated to be the one stop shop for cleaning a noprovision machine. If any command to clean up a machine has to be added, it must be added here. I only wrote the Checkbox side of business because that is what I know best